scripts/skills/perks/perk_rf_bloodlust.nut

```diff
@@ -22,82 +20,90 @@ this.perk_rf_bloodlust <- ::inherit("scripts/skills/skill", {
 
 	function isHidden()
 	{
-		return this.m.FatigueRecoveryStacks == 0;
+		return this.m.Stacks == 0;
 	}
 
 	function getTooltip()
 	{
-		local tooltip = this.skill.getTooltip();
-		tooltip.push({
+		local ret = this.skill.getTooltip();
+
+		local mult = 1.0 + (this.m.Stacks * this.m.MultPerStack);
+		ret.push({
 			id = 10,
 			type = "text",
+			icon = "ui/icons/resolve.png",
+			text = ::Reforged.Mod.Tooltips.parseString(::MSU.Text.colorizeMult(mult) + " increased [Resolve|Concept.Bravery]")
+		});
+		ret.push({
+			id = 11,
+			type = "text",
+			icon = "ui/icons/initiative.png",
+			text = ::Reforged.Mod.Tooltips.parseString(::MSU.Text.colorizeMult(mult) + " increased [Initiative|Concept.Initiative]")
+		});
+		ret.push({
+			id = 12,
+			type = "text",
 			icon = "ui/icons/fatigue.png",
-			text = "[color=" + ::Const.UI.Color.PositiveValue + "]+" + this.m.FatigueRecoveryStacks + "[/color] Fatigue Recovery on the next turn"
+			text = ::Reforged.Mod.Tooltips.parseString("Skills build up " + ::MSU.Text.colorizeMult(2.0 - mult, {InvertColor = true}) + " less [Fatigue Cost|Concept.Fatigue]")
```

Comment:
I think `2.0 - mult` is wrong here, same with `InvertColor = true`.
A mult of 0.25 (from 1 stack) reduces fatigue build up by 25%. But this line would describe that reduction as 75% less.

And from my understand it is correctly to have this kind of reduction colored in green, because it is good for the entity that receives this effect. 

I'm ok with keeping it red for weapon masteries for a bit, until we fix it in one go.
But I don't want us to continue that wrong coloring in other places. Finesse is also still wrong, but the reforged perks Skirmisher and Fluid Weapon had this fatigue cost reduction in colorGreen already.

Reply:
The calculation is correct. `2.0 - 1.25 = 0.75` which in `colorizeMult` will print `25%` in red. But we have invert color so it will print it in green.

Reply:
Ok then it will look correct.
But this still looks overcomplicated, is it necessary this way?

Reply:
Lol yes you are correct. I now realize that the way I did it was needlessly complicated. I'll change it.